Lib/argparse.py

```diff
@@ -1540,6 +1540,13 @@ def _get_positional_kwargs(self, dest, **kwargs):
             msg = _("'required' is an invalid argument for positionals")
             raise TypeError(msg)
 
+        # make sure 'store_true', 'store_false', or 'store_const' action
```

Comment:
This feels inelegant because we're hardcoding the names of individual actions. I'm not familiar enough with argparse to propose an alternative implementation though.

Reply:
@JelleZijlstra thank you for the review.

You say the change feels inelegant, because the names of individual actions are hardcoded. Do I understand correctly that you would prefer some named values or enumeration? Or did you mean something else?

I also am not a huge fan of using these strings directly, but this is the interface exposed by the `ArgumentParser` class and as such I think the names are committed to and won't change.

Anyway, I am open to all suggestions on how to improve this PR.

Reply:
I think that more elegant and general solution would be

* either add an optional boolean parameter for `registry()`
* or add an optional class attribute in `_StoreTrueAction` etc.

It can be tested in `add_argument()`, after testing that the action is a callable.

Reply:
Actually, such flag already exists. You can check if `nargs` is 0 to detect actions that do not consume arguments.